### DISCUSSION TITLE
Replacing archived fleetctl formula

### DISCUSSION THREAD

**Discussion by zwass** [Homebrew/discussions#4258] [Writing Formulae/Casks]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

- The original https://github.com/coreos/fleet repo was archived in 2020. The [formula](https://formulae.brew.sh/formula/fleetctl#default) was marked as archived in 2022.

- 0 installs on macOS or Linux in the last 90 days.

Would the Homebrew project be open to replacing this with the [fleetctl CLI](https://fleetdm.com/docs/using-fleet/fleetctl-cli) from https://github.com/fleetdm/fleet?

Thank you,
Zach Wasserman - CTO, Fleet

**Comment by SMillerDev** [comment]
Does coreos consider you their successor in this? Does upstream refer people to this new repo?

**Reply by zwass** [reply]
It's actually a completely unrelated (and archived) project that happens to have the same name. Is there a process defined for handling this kind of situation?

**Reply by SMillerDev** [reply]
Yes, the new project picks a different name for the formula because fleetctl is already taken

**Reply by zwass** [reply]
Is there any plan to prune abandoned projects out of homebrew at some point? We can make a new name and point folks to that, but it doesn't really seem like the most ideal user experience (for ours, or other abandoned and unused projects that share a namespace).

**Reply by SMillerDev** [reply]
Once it's deprecated for a year it can be disabled and once it's disabled for a year it'll be removed
